docs/source/contributing.rst

```diff
@@ -311,6 +322,11 @@ If you want to see what changes black will make, you can use::
 in-place.)
 
 
+Additionally, in some cases it is necessary to disable isort changing the
+order of imports. To do so you can add ``# isort: skip`` comments.
```

Comment:
Actually, as that page mentions `isort: split` should be preferred, `isort: skip` behaves weirdly.

Reply:
Changed in 14f5c12